qa/L0_backend_python/test.sh

```diff
@@ -370,6 +370,36 @@ and shared memory pages after starting triton equals to $current_num_pages \n***
     exit 1
 fi
 
+(cd lifecycle && bash -ex test.sh)
+if [ $? -ne 0 ]; then
+  RET=1
+fi
+
+(cd restart && bash -ex test.sh)
+if [ $? -ne 0 ]; then
+  RET=1
+fi
+
+(cd model_control && bash -ex test.sh)
+if [ $? -ne 0 ]; then
+  RET=1
+fi
+
+(cd examples && bash -ex test.sh)
+if [ $? -ne 0 ]; then
+  RET=1
+fi
+
+(cd argument_validation && bash -ex test.sh)
+if [ $? -ne 0 ]; then
+  RET=1
+fi
+
+(cd logging && bash -ex test.sh)
+if [ $? -ne 0 ]; then
+  RET=1
+fi
+
```

Comment:
@kthui It would also be nice to condense tests like this to something like:
```
subtests="lifecycle restart model_contrl examples ..."
for test in ${subtests}; do
  cd ${test} && bash -ex test.sh
  if [ $? -ne 0 ]; then
    echo "Subtest ${test} FAILED"
    RET=1
  fi
done
for 
```